### DISCUSSION TITLE
WHY DOES THE SERVER HAVE TO DISPLAY IT DIFFERENTLY FOR SOME PEOPLE!?!?

### DISCUSSION THREAD

**Discussion by joewinbpaul** [cssbattle/forum#338] [General]
I just don't know why:

![Screenshot (1438)](https://user-images.githubusercontent.com/102139966/213727260-3653421a-16fd-40e9-b9ec-cadecf791870.jpg)

It is a 100% match... 

BUT HOW!!!??? IT LITERALLY USES AN ARABIC LETTER (ݤ) !!!

And that's not the only one:

![Screenshot (1439)](https://user-images.githubusercontent.com/102139966/213727961-68ddb880-ac31-411b-996c-e6b42af0ccff.png)

Apparently this is also 100%..

**Comment by joewinbpaul** [comment]
Because of this, I have to use the CSSBattle Previewer every time I try Unicode...
<p>Take target 4 for example:
<p> This is how my computer displays it 👇👇👇 <p>

![Screenshot (1443)](https://user-images.githubusercontent.com/102139966/213731798-b2abde22-7c02-4ddf-9a5f-dce9cef1b350.png)

<p>And this is how the previewer sees it 👇👇👇<p>

![Screenshot (1442)](https://user-images.githubusercontent.com/102139966/213731973-9506a9de-cbc0-4a63-9f4f-9c4faf39eb4b.png)

**Reply by joewinbpaul** [reply]
And btw, it is the EXACT SAME code and is a 100% match.

**Reply by joewinbpaul** [reply]
And it is not the only target..

**Comment by aarthvaen** [comment]
As CSSBattle warns you every time you're loading / trying a Unicode-based solution:

> HTML entities are rendered differently in different systems and might not work as you expect

This has been explained couple times, so here's how it works:
> Submitted code solutions are rendered in a headless Chromium browser on a Ubuntu 18.04 server.
> 
> Your code is rendered as it is without any modification and the browser settings are all set on default.
> 
> There might be slight differences in how different browsers render your code in the game preview. Hence till the time we find a way a reliable way to normalize these differences, we recommend playing on Chrome, Chromium, Brave or Safari browser (these are the ones we have tested).
